Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,6 +25,10 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
     "runtimeConfig": {
         "gcServer": true,
         "gcConcurrent": false
+    },
+    "targetFramework": {
```

Comment:
Chatted with @davidfowl - He misunderstood this, we are doing this as discussed in this document.